test/e2e/redundancy/gateway_failover.go

```diff
@@ -7,42 +7,26 @@ import (
 	. "github.com/onsi/gomega"
 	"github.com/submariner-io/submariner/test/e2e/framework"
 	"github.com/submariner-io/submariner/test/e2e/tcp"
-	appsv1 "k8s.io/api/apps/v1"
 )
 
-var _ = PDescribe("[redundancy] Gateway fail-over tests", func() {
+var _ = Describe("[redundancy] Gateway fail-over tests", func() {
```

Comment:
@sridhargaddam  ^ can we change this one to PDescribe, and then we roll it back to Describe once we release the new subctl/operator ?

Reply:
Done